### PR TITLE
Put breadcrumbs behind feature switch

### PR DESCRIPTION
**Pull request by dcramer**
@getsentry/api @getsentry/ui

### CODE REVIEW

File: src/sentry/static/sentry/app/views/sharedGroupDetails/index.jsx [hunk 1]
```diff
@@ -95,7 +95,12 @@ const SharedGroupDetails = React.createClass({
             <div className="content">
               <SharedGroupHeader group={group} />
               <div className="group-overview">
-                <EventEntries group={group} event={evt} isShare={true} />
+                <EventEntries
+                  group={group}
+                  event={evt}
+                  orgId={group.project.organization.slug}
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/sharedGroupDetails/index.jsx]: we should try to get some tests in for this (i might take a stab at it) as this was missing required props

File: src/sentry/static/sentry/app/components/events/eventEntries.jsx [hunk 2]
```diff
@@ -99,8 +103,8 @@ const EventEntries = React.createClass({
         <EventTags
```
Comment (dcramer) [inline src/sentry/static/sentry/app/components/events/eventEntries.jsx]: also of note this should not link things in the public display

File: src/sentry/static/sentry/app/components/events/eventEntries.jsx [hunk 3]
```diff
@@ -99,8 +103,8 @@ const EventEntries = React.createClass({
...
           group={group}
           event={evt}
-          orgId={orgId}
-          projectId={projectId} />
+          orgId={this.props.orgId}
+          projectId={project.slug} />
```
Comment (mitsuhiko) [inline src/sentry/static/sentry/app/components/events/eventEntries.jsx]: projectId -> slug? If that's the case then i'm all for renaming that property.

Comment (dcramer) [inline src/sentry/static/sentry/app/components/events/eventEntries.jsx]: my goal when writing it was that we could transition without renaming, but I don tknow if that'll ever happen (e.g. ideally the project ID *is* the slug)

it's also orgId and projectId everywhere, so def not renaming as part of this

**Comment by mitsuhiko** [comment]
Looks good other than the id slug thing. I would add a comment because it looks so confusing unless we really have this like this everywhere. I did not notice that so far.

**Pull request merged**